examples/using-sqip/src/pages/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+const IndexPage = () => (
+  <div>
+    <h1>Gatsby SQIP Example</h1>
+    <p>@todo add description</p>
```

Comment:
Note to myself: Add some meta information and documentation for sqip to the example body